samples/MvvmSampleXF/MvvmSampleXF/MvvmSampleXF/Assets/docs/AsyncRelayCommand.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
I think it'd be cleaner if these files weren't duplicated, can we reference them here as a link?
I was also planning to do that for the UWP sample app eventually.

Reply:
I was hoping to reuse them as well. I ran into some issues with the component I am using to parse Markdown on mobile. It didn't like have `<` or `>` as text for a hyperlink so I made changes to the markdown that is specifically for mobile. I can always move these edited files to a shared location so all sample apps can use them instead though

Reply:
Ok these docs now live in the Core project and are added a links to the project heads. I made some small tweaks so it would work on mobile